client/landing/jetpack-cloud/components/scan-threats/index.tsx

```diff
@@ -173,6 +168,30 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 					/>
 				) ) }
 			</div>
+
+			{ error && (
+				<div className="scan-threats__error">
+					{ translate(
+						'The scanner was unable to check all files and errored before completion.{{lineBreak/}} Deal with the threats found above and run the {{runScan}}scan again{{/runScan}}. If the error persists, we are {{linkToSupport}}here to help{{/linkToSupport}}.',
+						{
+							components: {
+								lineBreak: <br />,
+								runScan: (
+									<Button className="scan-threats__run-scan-button" onClick={ dispatchScanRun } />
```

Comment:
We currently have the running of scan behind a feature flag. Since it is a feature that we might want to lunch without. 

Lets make create 2 different versions of the message one with the scan button and the other without.